### PR TITLE
[IMP] mrp: install quality_worksheet in setting

### PR DESCRIPTION
**Pull request by yhu-odoo**
Add a checkbox to install quality_worksheet in mrp setting, it will be
shown when quality_control is installed.

Task 2281266

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -53,6 +53,15 @@
                                     <div class="text-muted">
                                         Add quality checks to your work orders
                                     </div>
+                                    <div class="row mt-2" attrs="{'invisible': [('module_quality_control','=',False)]}">
+                                        <field name="module_quality_worksheet" widget="upgrade_boolean" class="col-lg-1 ml16 mr0"/>
+                                        <div class="col pl-0">
+                                            <label for="module_quality_worksheet"/>
+                                            <div class="text-muted">
+                                                Create costomizable worksheet for your quality check
```
Comment (Whenrow) [inline addons/mrp/views/res_config_settings_views.xml:61]:
```suggestion
                                                Create customizable worksheets for your quality checks
```

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo rebase-merge

File: addons/stock/views/res_config_settings_views.xml [hunk 2]
```diff
@@ -47,6 +47,27 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-12 col-lg-6 o_setting_box" id="quality_control">
```
Comment (Whenrow) [inline addons/stock/views/res_config_settings_views.xml:50]: I just saw this new quality setting uses the same id than the one in mrp. That means the Timer setting (related exclusively to workorders) is visible in the Stock settings

see https://github.com/odoo/enterprise/blob/3d4ed9ce80c0a9c3fec55bbd58967f5383c89dc2/mrp_workorder/views/res_config_settings_view.xml#L15 :)

Comment (yhu-odoo) [reply addons/stock/views/res_config_settings_views.xml:50]: Fixed at https://github.com/odoo/odoo/pull/63974

**Pull request closed without merging**